src/test/java/io/github/thebusybiscuit/slimefun4/core/guide/TestGuideOpening.java

```diff
@@ -90,8 +93,28 @@ void testOpenItemStack() throws InterruptedException {
     }
 
     @Test
-    @DisplayName("Test if the Slimefun Search can be opened from the History")
+    @DisplayName("Test if the Slimefun Search works with normal and colored querys")
     void testOpenSearch() throws InterruptedException {
```

Comment:
I like this test, let's name this
```suggestion
    void testOpenSearch_withColoredSearchTerm() throws InterruptedException {
```